### ISSUE TITLE
Feishin won't play next song in queue when using web player

### ISSUE THREAD

**Issue by ChekeredList71** [jeffvli/feishin#577]
## Expected Behavior

When there is a next element in the queue of Feishin, it should continue to the next song, when the current one ends.

## Current Behavior

Feishin jumps to the next song, when:
- I click on the artist's top 1 song, it ends then the top 2 starts playing*

Feishin doesn't jump to the next song, when:
- the second top song is over (started playback like in the example marked with *)
- any album is played (from the first song or any other point)
- music is started with the artist playbutton

Clearing browser cache doesn't help either.

## Steps to Reproduce

1. Switch to Web player

2. Start a song with any of the methods mentioned under the "Feishin doesn't jump to the next song, when:" title

3. Notice, how the song gets stuck at a timestamp one second before the song is over or at the end of the song. The pause button is still visible, Feishin shows the song as still playing.

![image](https://github.com/jeffvli/feishin/assets/66330496/70754f0b-677c-43a6-8f42-1c1ccb76b4d7)

## Possible Solution (Not obligatory)

On Debian 12 switching to MPV works. Any of the issues only appear with web audio player. 
However, this didn't solved the issue for #550 on Windows.

## Context

It's a big deal-breaker that Feishin won't play the next song, after all this is a must have for any music player. Luckily mpv solves the issue for me. 

## Your Environment

-   Application version: Feishin 0.6.1, AppImage
-   Operating System and version : Debian 12.2.0-14
-   Server : Navidrome 0.51.1
-   other:  mpv 0.35.1, FFmpeg 5.1.4-0+deb12u1

The issue appeared after updating Feishin from no idea what version. Previously I've been using no idea what player.

**Comment by kgarner7** [comment]
I am unable to reproduce this on any system (Windows, macOS, Linux) desktop or browser. Can you try a recent release (https://github.com/jeffvli/feishin/actions/runs/8549104144)?

**Comment by kgarner7** [comment]
Since there have been no updates to this issue, I'm going to go ahead and close this issue. If it is still relevant, please provide more details.

**Comment by ChekeredList71** [comment]
Apologies for the complete ghosting. I've just changed to MPV and forgot about it.

The issue is still there, now on Linux Mint, latest Feishin version. Even after I decreased the complexity of my setup. If you remember me from Discord, I had issues with my SMB share between my machine with ND and my NAS. Now I have ND as directly on my NAS (Docker).

As for the issue, it's still there. Take a look at this recording:
https://github.com/user-attachments/assets/6ce73255-92cd-4611-9e77-1b55b67553a8

Moreover, in 2 instances out of 4-5 tests, the second song actually started playing, but the player panel didn't show it. After I paused it and tried unpausing, it broke.

https://github.com/user-attachments/assets/4b5da906-096b-4b12-afd6-9b03a9fc159c
